pom.xml

```diff
@@ -27,7 +27,7 @@
     <revision>1.15.0</revision>
     <changelist>-SNAPSHOT</changelist>
     <java.level>8</java.level>
-    <jenkins.version>2.121.1</jenkins.version> <!-- Should be kept in sync with the version resulting from Dockerfile FROM statement-->
+    <jenkins.version>2.138.4</jenkins.version> <!-- Should be kept in sync with the version resulting from Dockerfile FROM statement-->
```

Comment:
Required for the workflow-api update. Current LTS is 2.164.2, not sure if you want to go all the way to that version though.

Reply:
So do you need to update some `Dockerfile` as the comment asks?

Reply:
The Dockerfile just [uses `FROM jenkins/jenkins:lts`](https://github.com/jenkinsci/blueocean-plugin/blob/d0f12cb922e608105a190ebd4126ff0ef4f3b9c7/Dockerfile#L10), and that version is currently 2.164.2. I wasn't sure if we want to update to such a new version, hence the comment. Perhaps the comment should just be deleted now that the Dockerfile uses the latest LTS.

Reply:
> Perhaps the comment in pom.xml should just be deleted now that the Dockerfile always tracks the latest LTS.

Or the `Dockerfile` should be made to pin a version. Depends on what it is being used for.

Reply:
I would however update the one in Jenkinsfile (but that looks like its already 2.138.4, so it doesn't matter)

Reply:
Ah good point, [this line](https://github.com/jenkinsci/blueocean-plugin/blob/d0f12cb922e608105a190ebd4126ff0ef4f3b9c7/Jenkinsfile#L43) needs to be deleted/or modified since we wouldn't be able to test on a baseline older than 2.138.4 anymore.

Reply:
Go ahead.

I did learn recently when i cleaned up the jenkinsfile, the weekly ath isn't run anymore anyways so no fears of breaking things

Reply:
Done in d5861ae.